src/ort_genai_c.h

```diff
@@ -44,6 +44,9 @@ typedef struct OgaTokenizerStream OgaTokenizerStream;
  */
 OGA_EXPORT const char* OGA_API_CALL OgaResultGetError(const OgaResult* result);
 
+OGA_EXPORT OgaResult* OGA_API_CALL OgaSetLogBool(const char* name, bool value);
+OGA_EXPORT OgaResult* OGA_API_CALL OgaSetLogString(const char* name, const char* value);
```

Comment:
>name [](http://example.com/codeflow?start=63&length=4)

list all the supported names here?

Reply:
We should put a link to the documentation